src/wovnio/wovnphp/custom_domain/CustomDomainLangUrlHandler.php

```diff
@@ -20,8 +20,8 @@ public static function changeToNewCustomDomainLang($absoluteUrl, $currentCustomD
                 "(${currentHostAndPath})". // 2. host and path
                 '((?:/|\?|#).*)?$' . // 3: other
                 '@';
-
-            return preg_replace($regex, "$1${newHostAndPath}$3", $absoluteUrl);
+            $replacement = '${1}' . "${newHostAndPath}" . '${3}';
```

Comment:
does this need to be in a variable though? Sounds like `"${1}${newHostAndPath}${3}"` would be equivalent?

Reply:
Ah it doesn't. I left it like that mostly for clarity.